data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml

```diff
@@ -0,0 +1,16 @@
+---
+- id: 788ad640-d9ec-452b-8a75-31b9361f15b4
+  name: Find Windows Record IDs
+  description: Given a PID, find the associated Windows Record IDs
+  tactic: detection
+  technique:
+    attack_id: x
+    name: x
```

Comment:
this is fine for now but we should brainstorm ways of improving this for the response abilities

Reply:
Do you mean improving the description, tactic, technique? Or something else?